adoc/release-2.1.0.adoc

```diff
@@ -0,0 +1,113 @@
+// Start attribute entry list (Do not edit here! Edit in entities.adoc)
+ifdef::env-github[]
+:suse: SUSE
+:current-year: 2020
+:product: {suse} Cloud Application Platform
+:version: 2.0
+:rn-url: https://www.suse.com/releasenotes
+:doc-url: https://documentation.suse.com/suse-cap/2
+:deployment-url: https://documentation.suse.com/suse-cap/2.0/single-html/cap-guides/#part-cap-deployment
+:caasp: {suse} Containers as a Service Platform
+:caaspa: {suse} CaaS Platform
+:ostack: OpenStack
+:cf: Cloud Foundry
+:kubecf: KubeCF
+:k8s: Kubernetes
+:scc: {suse} Customer Center
+:azure: Microsoft Azure
+:aks: Azure {k8s} Service
+:aksa: AKS
+:aws: Amazon Web Services
+:awsa: AWS
+:eks: Amazon Elastic Container Service for Kubernetes
+:eksa: Amazon EKS
+:gke: Google Kubernetes Engine
+:gkea: GKE
+:mysql: MySQL
+:mariadb: MariaDB
+:postgre: PostgreSQL
+:redis: Redis
+:mongo: MongoDB
+:ng: NGINX
+endif::[]
+// End attribute entry list
+
+[id='sec.2_1_0']
+=== Release 2.1.0, October 2020
+
+[id='sec.2_1_0.new']
+==== What Is New?
+* {kubecf} has been updated to version 2.5.8:
+** For a full list of features and fixes see https://github.com/cloudfoundry-incubator/kubecf/releases/tag/v2.5.8
+* {operator} has been updated to version 6.1.17:
+** For a full list of features and fixes see https://github.com/cloudfoundry-incubator/quarks-operator/releases/tag/v6.1.17
+* Stratos Console has been updated to version 4.1.0:
+** Configurable {ng} protocols and ciphers using Helm values
+** For a full list of features and fixes see https://github.com/SUSE/stratos/blob/master/CHANGELOG.md#410.
+
```

Comment:
As above, this must list all changes in Stratos since CAP 2.0.1

Reply:
See my comment above. Here, the situation is even simpler because it's all in a single file so if you just keep scrolling down, you can easily see all the changes since the last release. We would just be adding a bunch of links to different lines in that file...

Reply:
Fine for Stratos to have just one link so long as it's clear there has been more than one Stratos release since 2.0.1